worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -251,29 +256,30 @@ private MemoryManager(CelebornConf conf) {
         Utils.bytesToString(memoryShuffleStorageThreshold));
   }
 
+  /**
+   * Get current worker serving state. isPaused should be set true when trigger pause, and set false
+   * when trigger resume.
+   */
   public ServingState currentServingState() {
```

Comment:
What if we removed the comments from this method? Will this method be easier to understand?

Reply:
Sounds reasonable, done.